<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAToolClass</class>
 <widget class="QMainWindow" name="FAToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>移印机小工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>67</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>流控制</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>113</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbStopBits">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClrRcv">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清接收区</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBaudRate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>158</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>奇偶校验位</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teSendData">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;eeec0000fffcffff&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbSendHex">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbFlowControl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>204</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbSendASCII">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbRcvASCII">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>数据发送区</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbDataBits">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPortName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbParity">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbRcvHex">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>十六进制</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teReceiveData">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>数据接收区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetDefault">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复出厂设置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>设备类型（平台）</string>
    </property>
    <widget class="QRadioButton" name="rb0Sensors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0传感器</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb4Sensors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4传感器</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb3Sensors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>3传感器</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb2Sensors">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2传感器</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnInitBoardID">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>390</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化板ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClsBoardID">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除板ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRestart">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重启</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>印头传感器类型</string>
    </property>
    <widget class="QRadioButton" name="rbHeadSensorsNormalClose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>常闭（5V）</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbHeadSensorsNormalOpen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>常开（12V）</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>平台传感器类型</string>
    </property>
    <widget class="QRadioButton" name="rbPlatformSensorsNormalClose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>常闭（5V）</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbPlatformSensorsNormalOpen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>常开（12V）</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cbSendNewLine">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送新行</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbChangeSetupNow">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>直接更改设置（无需恢复出厂设置）</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbCurrentProgramNum">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>441</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前程序编号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbCurrentProgramNum">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>440</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_6</zorder>
   <zorder>label_3</zorder>
   <zorder>cmbStopBits</zorder>
   <zorder>label</zorder>
   <zorder>btnClrRcv</zorder>
   <zorder>cmbBaudRate</zorder>
   <zorder>label_4</zorder>
   <zorder>teSendData</zorder>
   <zorder>btnOpen</zorder>
   <zorder>rdbSendHex</zorder>
   <zorder>cmbFlowControl</zorder>
   <zorder>label_5</zorder>
   <zorder>rdbSendASCII</zorder>
   <zorder>rdbRcvASCII</zorder>
   <zorder>cmbDataBits</zorder>
   <zorder>cmbPortName</zorder>
   <zorder>btnSend</zorder>
   <zorder>cmbParity</zorder>
   <zorder>rdbRcvHex</zorder>
   <zorder>teReceiveData</zorder>
   <zorder>btnResetDefault</zorder>
   <zorder>btnInitBoardID</zorder>
   <zorder>btnClsBoardID</zorder>
   <zorder>btnRestart</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>cbSendNewLine</zorder>
   <zorder>cbChangeSetupNow</zorder>
   <zorder>lbCurrentProgramNum</zorder>
   <zorder>cmbCurrentProgramNum</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FATool.qrc"/>
 </resources>
 <connections/>
</ui>
